### PR TITLE
[MAINT] Prerequisite checks done

### PR DESCRIPTION
**Pull request by enver-bisevac**

### CODE REVIEW

**Review by davejohnston — APPROVED** [review]
This looks good to me.

File: src/evaluator.ts [hunk 1]
```diff
@@ -289,22 +298,92 @@ export class Evaluator {
...
+    }
+
+    return undefined;
+  }
+
+  boolVariation(
+    identifier: string,
+    target: Target,
+    defaultValue = false,
+    callback: Callback = undefined,
```
Comment (knagurski) [inline src/evaluator.ts:378]: If this is `undefined` by default, should `callback` be optional?
```suggestion
    callback?: Callback,
```

**Review by knagurski — APPROVED** [review]
Looks good, just a wee thing

File: src/evaluator.ts (hunk 1 above)
Comment (enver-bisevac) [reply src/evaluator.ts]: Done

**Pull request merged**